src/catalog/postgres/builder.cpp

```diff
@@ -758,11 +760,11 @@ IndexSchema Builder::GetProcNameIndexSchema(db_oid_t db) {
   std::vector<IndexSchema::Column> columns;
 
   columns.emplace_back("pronamespace", type::TypeId::INTEGER, false,
-                       parser::ColumnValueExpression(db, PRO_TABLE_OID, PRONAMESPACE_COL_OID));
+                       parser::ColumnValueExpression(db, PgProc::PRO_TABLE_OID, PgProc::PRONAMESPACE_COL_OID));
   columns.back().SetOid(indexkeycol_oid_t(1));
 
   columns.emplace_back("proname", type::TypeId::VARCHAR, MAX_NAME_LENGTH, false,
-                       parser::ColumnValueExpression(db, PRO_TABLE_OID, PRONAME_COL_OID));
+                       parser::ColumnValueExpression(db, PgProc::PRO_TABLE_OID, PgProc::PRONAME_COL_OID));
   columns.back().SetOid(indexkeycol_oid_t(2));
 
   // Non-Unique, not primary
```

Comment:
This probably needs a more detailed comment as it violates expectations established by every other table in the catalog (i.e., that a namespace OID and name gets at most one result).  I also wonder if there is a way to add `PROARGTYPES` into the index to make it unique and simultaneously enforce the constraint that you cannot overload with the same argument signature.

Reply:
Any follow-up on this one?

Reply:
Inclined to punt it into #1361.